uui-core/src/hooks/useVirtualList/useVirtualList.ts

```diff
@@ -53,6 +57,20 @@ export function useVirtualList<List extends HTMLElement = any, ScrollContainer e
         rowsSelector,
     ]);
 
+    useLayoutEffectSafeForSsr(() => {
```

Comment:
Can we wrap all this with if (__DEV__), to throw away this some of this code in prod builds?